src/docs/cmdstan-guide/installation.tex

```diff
@@ -10,18 +10,17 @@ \chapter{Installation and Compatibility}\label{install.appendix}
 
 \section{Operating System}
 
-\CmdStan is written in portable \Cpp without {\Cpp}11 features, as are the
+\CmdStan is written in portable \Cpp with {\Cpp}11 and {\Cpp}14 features, as are the
 libraries on which it depends.  Therefore, \CmdStan should run on any machine
-for which a suitable \Cpp compiler is available.  In practice, \CmdStan,
-like the Boost and Eigen libraries on which it depends, is very hard
-on the compiler and linker.
+for which a suitable \Cpp compiler supporting C++1y or C++14 features is available. In practice, \CmdStan,
+like the Boost and Eigen libraries on which it depends, is very hard on the compiler and linker.
 
 \CmdStan has been tested on the following operating systems.
 %
 \begin{itemize}
-\item Linux (Debian, Ubuntu, Red Hat)
+\item Linux (Debian, Ubuntu)
 \item Mac OS X (from 10.6 ``Snow Leopard''  through 10.14 ``Mojave'')
```

Comment:
What is the Mac OS X version on the test machines. I can probably add Catalina here right?

Reply:
There are two, neither are on Catalina. I think one is Snow Leopard and one Mojave.

Reply:
Leaving as is then. Thanks for reviewing.